BraveRewardsUI/Ads/AdsViewController.swift

```diff
@@ -41,9 +41,14 @@ public class AdsViewController: UIViewController {
       $0.leading.greaterThanOrEqualTo(view).inset(8)
       $0.trailing.lessThanOrEqualTo(view).inset(8)
       $0.centerX.equalTo(view)
-      $0.width.equalTo(400).priority(.high)
```

Comment:
Not sure why this was hardcoded to 400 in the first place.
Seems to work fine with `view`.

Reply:
It was so that on larger screens it maxed out at 400pt's